core/templates/dev/head/pages/about.html

```diff
@@ -24,192 +24,81 @@
-      </p>
-
-      <strong>Learn</strong>
-      <p>
-        Oppia recreates the experience of one-on-one tutoring. It adapts to your
-        needs and lets you learn in the best way possible &mdash; by doing.
-      </p>
+  <div class="about-header">
+    <h1>Be Part of a Community of Learners</h1>
+  </div>
 
-      <a class="oppia-about-anchor" name="teach"></a>
-      <strong>Teach</strong>
-      <p>
-        Oppia enables you to share your knowledge with people worldwide. Anyone can create
-        and contribute to an exploration. Find out more in our <a href="/participate">participation playbook</a>.
-      </p>
+  <div class="oppia-page-card about-content">
+    <div class="about-nav oppia-profile-tabs md-tab">
+      <ul class="about-tab-links">
+        <li class="oppia-profile-tabs about-active-tab">
+          <a href="about-tab-1">About Oppia</a>
+        </li>
+        <li class="oppia-profile-tabs">
+          <a href="about-tab-2">The Oppia Foundation</a>
+        </li>
+        <li class="oppia-profile-tabs">
+          <a href="about-tab-3">Credits</a>
+        </li>
+      </ul>
+    </div> <!-- /about-nav -->
 
-      <a class="oppia-about-anchor" name="contact"></a>
-      <h3 style="margin-top: 40px;">Contacting Us</h3>
+  <div class="about-tab-container">
+    <div class="about-tab-1 about-tab-content about-visible-content">
+      <div class="about-card-content-wide about-card-content-narrow">
+        <h2>
+          Our Mission<br>
+          To help anyone learn anything they want in an effective and
+          enjoyable way.
+        </h2>
+        <p>
+          Oppia means "learn" in Finnish, and this site is all about learning and
+          sharing knowledge! Oppia is a free, open-source learning platform that
+          offers interactive learning modules that recreate the power of 1-on-1 tutoring.
+          Our unique technology allows anyone to create these learning modules, or,
+          as we call them, explorations, to share their knowledge in the most effective and
+          efficient way. Since these explorations adapt to the needs and the current knowledge
+          of learners, Oppia provides an enriched learning experience. Oppia's explorations cover
+          a variey of subjects, like mathematics, biology, and programming, and are offered
+          in various languages made by unique individuals like you.
+        </p>
+      </div><!-- /MISSION about-card-content-wide about-card-content-narrow -->
+    </div><!-- /tab -->
 
+  <div class="about-tab-2 about-tab-content">
+    <div class="about-card-content-wide about-card-content-narrow">
+      <h2>The Oppia Foundation</h2>
       <p>
-        <strong>For site questions:</strong> Post to our <a href="{{SITE_FORUM_URL}}">forum</a>.
+        The Oppia website and source code are supported by the Oppia Foundation,
+        a non-profit public benefit corporation of the State of California.
```

Comment:
Note that this text has been updated recently -- please see the latest version in develop, and modify accordingly. Thanks!